k8s/upgrades/0.9.0-1.0.0/claim_blockdevices.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
+
+###############################################################################
+# STEP 1: Get all block devices present on the cluster and corresponding disk #
+# STEP 2: Create block device claim to claim corresponding block device       #
+# STEP 3: Patch SPC to stop reconciliation                                    #
+#                                                                             #
+###############################################################################
+function error_msg() {
+    echo "Pre-upgrade step failed. Please make sure that pre-upgrade should be successful before going to next step"
+}
+
+function usage() {
+    echo
+    echo "Usage:"
+    echo
+    echo "$0 <openebs-namespace>"
+    echo
+    echo "  <openebs-namespace> Get the namespace where openebs setup is running"
```

Comment:
change message to `Namespace in which openebs control plane components are installed`

Reply:
Updated the message